bitnami/redis-cluster/7.0/debian-11/rootfs/opt/bitnami/scripts/libredis.sh

```diff
@@ -402,6 +406,14 @@ redis_configure_default() {
         # Enable AOF https://redis.io/topics/persistence#append-only-file
         # Leave default fsync (every second)
         redis_conf_set appendonly "${REDIS_AOF_ENABLED}"
+
+        if [ "${REDIS_RDB_ENABLED}" == "yes" ];then
+        #The value stored in $i here is the number of seconds and times of save rules in redis rdb mode
+            for i in ${REDIS_RDB_POLICY};do
+                redis_conf_set save "${i//#/ }"
+            done
+        fi
```

Comment:
I think we can remove the `REDIS_RDB_ENABLED` variable, taking less memory space for the environment and set only `REDIS_RDB_POLICY`. I suggest you set an if condition to disable this if the variable is not present, default is `save ""`. Please apply this to Redis 6.2 as well.
```suggestion
        if [[ -n "$REDIS_RDB_POLICY" ]]; then
          #The value stored in $i here is the number of seconds and times of save rules in redis rdb mode
          for i in ${REDIS_RDB_POLICY};do
            redis_conf_set save "${i//#/ }"
          done
        else
            redis_conf_set save ""
        fi
```